File: 2 D Array/Matrix Addition/matrixAddition.java
// Program to compute the addition of two matrices

import java.util.*;

public class matrixAddition 
{
    public static void main(String[] args)
    {
        try(Scanner sc = new Scanner(System.in))
        {
            //Scanning the size of the matrix
            System.out.print("Enter the size of the matrix: ");
            int size = sc.nextInt();

            int matrixOne[][] = new int[size][size];
            // Scanning the matrix I
            System.out.println("Enter the elements of matrix I: ");
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    System.out.print("Element ["+ i +"]["+ j +"]: ");
                    matrixOne[i][j] = sc.nextInt();
                }
            }

            int matrixTwo[][] = new int[size][size];
            // Scanning the matrix II
            System.out.println("Enter the elements of matrix II: ");
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    System.out.print("Element ["+ i +"]["+ j +"]: ");
                    matrixTwo[i][j] = sc.nextInt();
                }
            }

            // Printing matrix I
            System.out.println("Matrix I: ");
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    System.out.print(matrixOne[i][j] + " ");
                }
                System.out.println();
            }

            // Printing matrix II
            System.out.println("Matrix II: ");
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    System.out.print(matrixTwo[i][j] + " ");
                }
                System.out.println();
            }

            int sum[][] = new int[size][size];
            //Computing the addition
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    sum[i][j] = matrixOne[i][j] + matrixTwo[i][j];
                }
            }

            // Printing the sum
            System.out.println("Addtion of matrices: ");
            for(int i=0; i<size; i++)
            {
                for(int j=0; j<size; j++)
                {
                    System.out.print(sum[i][j] + " ");
                }
                System.out.println();
            }
        }
    }    
}

File: 2 D Array/Matrix Addition/matrixAddition.py
size = int(input("Enter the size of the matrices: "))

# Scanning matrix I
print("Enter the elements of Matix I: ")
matrixOne = []
for i in range(size):
    a = []
    for j in range(size):
        a.append(int(input(f"Element [{i}][{j}]: ")))
    matrixOne.append(a)

# Scanning matrix II
print("Enter the elements of Matix I: ")
matrixTwo = []
for i in range(size):
    b = []
    for j in range(size):
        b.append(int(input(f"Element [{i}][{j}]: ")))
    matrixTwo.append(b)

# Printing matix I
print()
print("Matrix I: ")
for i in range(size):
    for j in range(size):
        print(matrixOne[i][j], end = " ")
    print()

# Printing matrix II
print()
print("Matrix II: ")
for i in range(size):
    for j in range(size):
        print(matrixTwo[i][j], end = " ")
    print()

# Computing addition of matrices
Addition = []
for i in range(size):
    c = []
    for j in range(size):
        c.append(matrixOne[i][j] + matrixTwo[i][j])
    Addition.append(c)

# Printing addition of matrices
print()
print("Addition: ")
for i in range(size):
    for j in range(size):
        print(Addition[i][j], end = " ")
    print()
